e2e/cypress/integration/api/iam/projects_api.spec.ts

```diff
@@ -8,96 +8,57 @@ describeIfIAMV2p1('projects API', () => {
     status: string;
   }
 
-  describe('project graveyarding and deletion', () => {
-    const projectID = `${cypressPrefix}-to-delete-${Cypress.moment().format('MMDDYYhhmm')}`;
-    before(() => {
-      cy.cleanupV2IAMObjectsByIDPrefixes(cypressPrefix, ['projects', 'policies']);
-    });
-
-    it('gives a 404 when the project does not exist', () => {
-      cy.request({
-        headers: { 'api-token': Cypress.env('ADMIN_TOKEN') },
-        url: '/apis/iam/v2/projects/not_found',
-        failOnStatusCode: false
-      }).then((response) => {
-        expect(response.status).to.equal(404);
-      });
-    });
-
-    it('graveyards and then deletes a project that exists', () => {
-      cy.request({
-        headers: { 'api-token': Cypress.env('ADMIN_TOKEN') },
-        url: '/apis/iam/v2/projects',
-        method: 'POST',
-        body: {
-          id: projectID,
-          name: 'to delete'
-        }
-      });
-
-      cy.request({
-        headers: { 'api-token': Cypress.env('ADMIN_TOKEN') },
-        url: `/apis/iam/v2/projects/${projectID}`,
-        method: 'DELETE'
-      });
-
-      waitUntilProjectNotInGraveyard(projectID, 100);
-
-      cy.request({
-        headers: { 'api-token': Cypress.env('ADMIN_TOKEN') },
-        url: '/apis/iam/v2/projects',
-        method: 'POST',
-        body: {
-          id: projectID,
-          name: 'will fail'
-        },
-        failOnStatusCode: false
-      }).then((response) => {
-        expect(response.status).to.equal(409);
-      });
-      waitUntilProjectNotInGraveyard(projectID, 100);
-    });
+  const avengersProject = {
+    id: `${cypressPrefix}-project1-${Cypress.moment().format('MMDDYYhhmm')}`,
+    name: 'Test Avengers Project'
+  };
+
+  const xmenProject = {
+    id: `${cypressPrefix}-project2-${Cypress.moment().format('MMDDYYhhmm')}`,
+    name: 'Test X-men Project'
+  };
+
+  const avengersRule = {
```

Comment:
Consider making these tests even more robust by using "real" objects. Add the type here which ensures:
1. you have all properties of an object (making it more realistic);
2. you have not misspelled any property names; and
3. you have not assigned values of the wrong type.
```suggestion
  const avengersRule: Rule = {
```
To use this you need to add an import at top:
```
import { Rule } from '../../../../../components/automate-ui/src/app/entities/rules/rule.model';
```

Reply:
sounds like a good idea! though i'm curious if that nested directory location works on our buildkite machines

Reply:
True, it might not. In truth, I debated about including a reference to another project. It is probably the lesser of two evils to make a copy of `Rule` within the `e2e` project.